### PR TITLE
[StreamAnalytics] Remove Experimental

### PR DESCRIPTION
**Pull request by ZengTaoxu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

closed: https://github.com/Azure/azure-cli-extensions/issues/6499

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
StreamAnalytics

**Comment by jsntcy** [comment]
@ZengTaoxu, please update history notes and version as well.

File: src/stream-analytics/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.3
```
Comment (jsntcy) [inline src/stream-analytics/HISTORY.rst:6]:
```suggestion
1.0.0
```

File: src/stream-analytics/setup.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 from setuptools import setup, find_packages
 
 # HISTORY.rst entry.
-VERSION = '0.1.2'
+VERSION = '0.1.3'
```
Comment (jsntcy) [inline src/stream-analytics/setup.py:13]:
```suggestion
VERSION = '1.0.0'
```

File: src/stream-analytics/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.0
++++++++++++++++
+* Remove Experimental
```
Comment (jsntcy) [inline src/stream-analytics/HISTORY.rst:8]:
```suggestion
* Make extension as GA.
```

**Pull request merged**